Jenkinsfile_parameterized

```diff
@@ -32,12 +32,11 @@ def secrets = [
         secret('idam-secret', 'IA_IDAM_SECRET'),
         secret('idam-redirect-uri', 'IA_IDAM_REDIRECT_URI'),
         secret('s2s-secret', 'IA_S2S_SECRET'),
-        secret('s2s-microservice', 'IA_S2S_MICROSERVICE'),
         secret('ccd-gw-url', 'CCD_GW_URL'),
         secret('dm-url', 'DM_URL'),
         secret('idam-url', 'IDAM_URL'),
-        secret('s2s-url', 'S2S_URL')
-
+        secret('s2s-url', 'S2S_URL'),
```

Comment:
this should be replaced by
```
env.S2S_URL = "http://rpe-service-auth-provider-aat.service.core-compute-aat.internal"
```
because you are still reading them from vault

Reply:
other urls as well

Reply:
Replaced all urls here